runtime/src/bank.rs

```diff
@@ -952,10 +954,6 @@ pub(super) enum RewardInterval {
 }
 
 impl Bank {
-    pub fn new_for_benches(genesis_config: &GenesisConfig) -> Self {
-        Self::new_with_paths_for_benches(genesis_config, Vec::new())
-    }
-
     /// Intended for use by tests only.
     /// create new bank with the given configs.
     pub fn new_with_runtime_config_for_tests(
```

Comment:
I guess it doesn't fit the title, but shouldn't `new_with_runtime_config_for_tests` also be either test or dcou?